composer.json

```diff
@@ -1,28 +1,41 @@
 {
     "name": "pantheon-systems/pantheon_advanced_page_cache",
     "type": "drupal-module",
-    "description": "Pantheon Advanced Page Cache",
+    "description": "Drupal Integration for the Pantheon Advanced Page Cache. Code mirror at drupal/pantheon_advanced_page_cache.",
     "license": "MIT",
     "require": {
-        "php": ">=7.4"
+        "php": ">=7.4",
+        "symfony/http-foundation": "^6 || ^7",
```

Comment:
Do we _really_ need this package? Can't we rely on Drupal to get them? Aren't they part of Drupal core? Talking about `symfony/http-foundation` and `guzzlehttp/psr7`

The least Drupal core packages we require the best for us to reduce maintenance issues here

Reply:
Because it works with so many different versions of drupal, I thought we couldn't be sure it was a declared dependency by the host CMS.